mangaki/mangaki/views.py

```diff
@@ -383,56 +385,107 @@ def get_context_data(self, **kwargs):
         return context
 
 
-def get_profile(request, username):
-    user = get_object_or_404(User, username=username)
-    is_shared = user.profile.is_shared
+def remove_all_anon_ratings(request):
+    clear_anonymous_ratings(request.session)
+    return redirect('home')
+
+
+def get_named_profile(request, username):
+    return get_profile(request, username)
+
+
+def get_my_profile(request):
```

Comment:
I think this should be identical to `get_profile` with `username=None` (i.e. why do we need a separate function for the `if request.user.is_authenticated` instead of just having it in `get_profile`?).